code/messaging/src/main/java/com/adobe/marketing/mobile/aepcomposeui/state/LargeImageCardUIState.kt

```diff
@@ -0,0 +1,23 @@
+/*
+  Copyright 2025 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+*/
+
+package com.adobe.marketing.mobile.aepcomposeui.state
+
+/**
+ * Class representing the UI state of a Large Image template card.
+ *
+ * @property dismissed Indicates whether the card has been dismissed.
+ * @property displayed Indicates whether the card is currently displayed.
+ */
+data class LargeImageCardUIState(
+    override val dismissed: Boolean = false,
+    override val displayed: Boolean = false
+) : AepCardUIState()
```

Comment:
nit: 
```
data class LargeImageCardUIState(
    override val dismissed: Boolean,
    override val displayed: Boolean
) : AepCardUIState(dismissed, displayed)
```

Reply:
If the initial values are not provided here, I will have to pass it every time a new reference of it is created. Since it doesn't inherit the initial values defined in the parent `AepCardUIState` I removed those and retained the initial values here. Let me know if it makes more sense to explicitly specify these each time an instance is created.